packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -1005,6 +1005,12 @@ class ModalBottomSheetRoute<T> extends PopupRoute<T> {
 
   final ValueNotifier<EdgeInsets> _clipDetailsNotifier = ValueNotifier<EdgeInsets>(EdgeInsets.zero);
 
+  @override
+  void didAdd() {
```

Comment:
I think this should be in `dispose` instead. The didAdd is called when the bottomsheet is about to become visible, and we definitrly want the `_clipDetailsNotifier` to stay alive after that.

Reply:
thank you!
something flipped in outocomplete :)